SingularityService/src/main/java/com/hubspot/singularity/resources/DeployResource.java

```diff
@@ -134,4 +149,33 @@ public SingularityRequestParent cancelDeploy(
 
     return fillEntireRequest(requestWithState);
   }
+
+  @POST
+  @Path("/deploy/{deployId}/request/{requestId}")
```

Comment:
ordering here feels a little awkward -- i understand that you're following the pattern of our other resources, but it might make more sense flip it to `/deploys/request/{requestId}/deploy/{deployId}`